libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -687,6 +781,12 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
         }
     }
 
+    button.is-normal {
+        .gd-button-text {
+            color: #6d7680;
```

Comment:
We can reuse the variable defined above for this, can't we?

Reply:
I have changed to defined value thanks